src/index.js

```diff
@@ -21,6 +22,12 @@ exports.createRouter = (config = {}) => {
   router.use(bodyParser.json({}))
   router.post('/send', (req, res, next) => {
     const { body } = req
+    const token = body.token
+    if( token !== null && token !== env('TOKEN')){
```

Comment:
If the token property is not setted, it will be `undefined`, not `null`, then the first check always is true, however the second check catches it. It could be improved using `token != null` (it checks if it's `undefined` or `null`, both cases)

Reply:
That's true, I just to do this fix (I always pass a token :D). 

Thanks very much!